package.json

```diff
@@ -38,7 +38,6 @@
     "cross-spawn": "^2.1.5",
     "jquery": "^2.2.0",
```

Comment:
Cool, that is what I thought. Probably makes more sense to move to `devDependencies`.